search-app/resources/templates/eosdis-landing-links.html

```diff
@@ -0,0 +1,9 @@
+{% extends "templates/base.html" %}
+
+{% block title %}CMR Search - Landing Pages for EOSDIS Collections{% endblock %}
```

Comment:
We should double-check whether the "Landing page" terminology is fine to use. I remember awhile back we were told to not call our HTML rendering of collection metadata as a "Collection Landing Page". Let's ask Dana and ESDIS.